events/janus_wsevh.c

```diff
@@ -69,13 +69,21 @@ janus_eventhandler *create(void) {
 	return &janus_wsevh;
 }
 
-
 /* Useful stuff */
 static volatile gint initialized = 0, stopping = 0;
 static GThread *ws_thread, *handler_thread;
 static void *janus_wsevh_thread(void *data);
 static void *janus_wsevh_handler(void *data);
 
+/* Connection related helper methods */
+static void janus_wsevh_schedule_connect_attempt(void);
+static void janus_wsevh_calculate_reconnect_delay_on_fail(void);
+/* lws_sorted_usec_list_t is defined starting with lws 3.2 */
+#if !(((LWS_LIBRARY_VERSION_MAJOR == 3 && LWS_LIBRARY_VERSION_MINOR >= 2) || LWS_LIBRARY_VERSION_MAJOR >= 4))
+	#define lws_sorted_usec_list_t void
+#endif
+static void janus_wsevh_connect_attempt(lws_sorted_usec_list_t* sul);
```

Comment:
Per code style, the `*` should be next to the pointer, not the type:

```
lws_sorted_usec_list_t *sul
```

Reply:
ack